Makefile-docker

```diff
@@ -87,7 +87,12 @@ populate_data: ## populate a new database
 	# These add-ons are specifically useful for the addons-frontend
 	# homepage. You may have to re-run this, in case the data there
 	# changes.
-	$(PYTHON_COMMAND) manage.py generate_default_addons_for_frontend
+	@if test "$(UITEST_FXA_EMAIL)" != ""; then \
+		echo 'UITEST_FXA_EMAIL is present, running generate_default_addons_for_frontend';\
+        $(PYTHON_COMMAND) manage.py generate_default_addons_for_frontend;\
```

Comment:
wait... don't we _need_ this or the local frontend is entirely empty so fails?  It's the thing that populates the database/ES with enough addons.

Reply:
It's just the fxa account creation part that can be skipped.

Reply:
We already generate a bunch of add-ons and themes a few lines above this one, so I'd say no ?

Reply:
That was added for https://github.com/mozilla/addons-server/issues/9080 ... but I doubt it's still up to date anyway, so I don't know ? I must admit I usually don't particularly care about the homepage content on my local env...

Reply:
it's the command that creates the promoted addons and collections.  IIRC frontend fails badly when nothing is available.  I don't care that much about the homepage being fully populated, though I feel if it raises an error and doesn't work at all we've just moved where a contributor would be blocked to a different part in the process.

Can we just bypass the part of the command that creates the user on FxA instead?  (skip `self.fxa_id = self._create_fxa_user()` if `UITEST_FXA_EMAIL` is missing or something)?

Reply:
Yeah, I initially started with that and then starting to question why we even called this command in the first place... I'll go back to having the condition in the command.